R/load.r

```diff
@@ -92,10 +95,10 @@
 #' load_all("./", export_all = FALSE)
 #' }
 #' @export
-load_all <- function(path = ".", reset = TRUE, recompile = FALSE,
+load_all <- function(path = ".", reset = TRUE, compile = NA,
```

Comment:
I think it might be more self-documenting to make the default `pkgbuild::needs_compile()`, e.g.

```suggestion
load_all <- function(path = ".", reset = TRUE, compile = pkgbuild::needs_compile(),
```